test/app.e2e-spec.ts

```
@@ -16,10 +62,34 @@ describe('AppController (e2e)', () => {
       .expect(200)
-      .expect('Hello World!');
+      .expect('Content-Type', /text\/plain/)
+      .expect((res) => {
+        expect(res.text).toContain('schema');
+      });
+  });
+
+  it('returns 404 for unknown projects', () => {
+    return request(app.getHttpServer())
+      .get('/supergraph/unknown')
+      .expect(404);
```

ESLint

[error] Replace `⏎······.get('/supergraph/unknown')⏎······` with `.get('/supergraph/unknown')`

(prettier/prettier)